lib/install/config/webpacker.yml

```diff
@@ -34,6 +34,7 @@ development:
     host: localhost
     port: 8080
     https: false
+    hot_reloading: false
```

Comment:
@gauravtiwari @dhh Let's see if we can come to a quick agreement on the naming of the ENV and YML values.